### DISCUSSION TITLE
Can we have an example for directional variogram estimation and Kriging?

### DISCUSSION THREAD

**Discussion by flydream0428** [GeoStat-Framework/GSTools#264] [Ideas]
Hi, can someone add a section to the document to show how to estimate a directional variogram using some unstructured data? 
Once the directional variogram is estimated, how to use this to do interpolation?

**Comment by MuellerSeb** [comment]
That would be of course very nice. Do you have any specific ideas for such an example?

We still have the problem, that the estimation of rotation angles can't be done automatically with GSTools, so you need to know/guess the rotation angles beforehand. Maybe a 2D example would be best here, since we only have one rotation angle in the xy plane.

We could also come up with a simple script to find the main-direction by estimating the variogram in a number of directions, fitting a specified model to it and then selecting the direction with the biggest correlation length.

Maybe you give a suggestion. :upside_down_face:
